Packs/Gatewatcher/Integrations/GCenterv102/GCenterv102.yml

```
@@ -0,0 +1,1119 @@
+      description: The Probability It Is Obfuscated
+      type: Number
+    - contextPath: GCenter.Alerts.Single.powershell.scores.analysis
+      description: The Powershell Analysis Score
+      type: Number
+    - contextPath: GCenter.Alerts.Single.shellcode.file_id
+      description: The File Id
+      type: String
+    - contextPath: GCenter.Alerts.Single.shellcode.encodings.name
+      description: The Name Of The Encoding
+      type: String
+    - contextPath: GCenter.Alerts.Single.shellcode.encodings.count
+      description: The Number Of The Encoding Elements
+      type: Number
+    - contextPath: GCenter.Alerts.Single.shellcode.calls.call
+      description: The Name Of The Call Of The Alert
+      type: String
+    - contextPath: GCenter.Alerts.Single.shellcode.calls.args
+      description: The Argument Used For The Call
+      type: String
+    - contextPath: GCenter.Alerts.Single.shellcode.calls.ret
+      description: The Retention Of The Call
+      type: String
+    - contextPath: GCenter.Alerts.Single.shellcode.calls.index
+      description: The Call Index
+      type: Number
+    - contextPath: GCenter.Alerts.Single.malware.analyzed_clean
+      description: Number Of Engines That Returned A Clean Status
+      type: Number
+    - contextPath: GCenter.Alerts.Single.malware.analyzed_infected
+      description: Number Of Engines That Returned An Infected Status
+      type: Number
+    - contextPath: GCenter.Alerts.Single.malware.analyzed_suspicious
+      description: Number Of Engines That Returned A Suspicious Status
+      type: Number
+    - contextPath: GCenter.Alerts.Single.malware.analyzed_other
+      description: Number Of Engines That Returned Other Statuses
+      type: Number
+    - contextPath: GCenter.Alerts.Single.malware.analyzed_error
+      description: Number Of Engines That Failed To Analyze The File
+      type: Number
+    - contextPath: GCenter.Alerts.Single.malware.code
+      description: The Global Code Result
+      type: Number
+    - contextPath: GCenter.Alerts.Single.malware.def_time
+      description: 'When The Last Engines Have Ended The Scan '
+      type: Date
+    - contextPath: GCenter.Alerts.Single.malware.scan_time
+      description: The Scan Time In Ms.
+      type: Number
+    - contextPath: GCenter.Alerts.Single.malware.threats_found
+      description: The Threats Found By The Engines
+      type: String
+    - contextPath: GCenter.Alerts.Single.malware.reporting_token
+      description: The Reporting Token Returned By The Gbox.
+      type: String
+    - contextPath: GCenter.Alerts.Single.malware.engines_report.0.id
+      description: The Hash Pf The Engine
+      type: String
+    - contextPath: GCenter.Alerts.Single.malware.engines_report.0.threat_details
+      description: The Threat Found By The Engine
+      type: String
+    - contextPath: GCenter.Alerts.Single.malware.engines_report.0.scan_result
+      description: Analysis Result
+      type: String
```

Please remove all the below engine outputs - I think it would be better to output all the engines details in one list here as well. 
```suggestion
    - contextPath: GCenter.Alerts.Single.malware.engines_report
      description: The Engines Report
      type: List
```
Please update the readme file with this change as well.

---

As discussed on slack - you decided to keep the outputs of the 'engines_report' as they are here (a dictionary with the 15 engines' entries). I recommend that you'll change that in the next version of the integration :)